crates/holochain_cascade/src/lib.rs

```diff
@@ -764,15 +764,15 @@ impl CascadeImpl {
     }
 
     /// Gets links from the DHT or cache depending on its metadata.
-    // The default behavior is to skip deleted or replaced entries.
+    /// Deleted or replaced entries are skipped.
```

Comment:
I don't think there is any non-default behavior that could be specified and that *would* get deleted or replaced entries. That's what `get_link_details()` would be here for but that's a separate call altogether.

Reply:
No, I'm wrong about `get_link_details()` being here for that. `get_link_details()` is about getting deleted *links* also. Still, I don't think there is a way to explicitly get deleted or replaced entries with the current get_links API.

Reply:
Agree with both assessments. It's not default behavior here, it's the only behavior. Get details does return deleted links for whatever reason. I guess you might want to see if somebody has deleted all your links and restore them or something? But I think this is a good doc update here.